hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.js

```diff
@@ -0,0 +1,70 @@
+import React, { PureComponent } from 'react'
+import styles from './Navbar.module.scss';
+import { Link } from 'react-router-dom';
+import { ReactComponent as Logo } from './../../../logo.svg';
+import { Home, ShoppingCart, Notifications, ListAlt, AccountCircle, InsertChartOutlined } from '@material-ui/icons';
+import { ReactComponent as Inventory } from './../../../assets/images/icons/Hardware.svg';
+import { IconButton, AppBar, Toolbar, Typography } from '@material-ui/core';
+
+export default class Navbar extends PureComponent {
```

Comment:
We decided on using functional components instead of class-based components in our last meeting for consistency. So, 
```javascript
const Navbar = () => (
    // everything inside the return starting on line 12
);
export default Navbar;
```